json_to_html.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Maybe it's better to be put to `/ci/` or something? I don't think this file should be in the root.

Reply:
Moved to ```ci``` folder.

Reply:
Maybe we need to exclude it from `setup.py`. As well as `docs/`. They are not modules, though.

Reply:
Added these directories by analogy with ```tests```

Reply:
> Maybe we need to exclude it from `setup.py`. As well as `docs/`. Even though they are not modules yet.

FWIW, I think we should move to a `src`-based layout (i.e., create an `src` directory, move `datumaro` there and modify `setup.py` to only look for packages there) to prevent unwanted files from being included in the distribution by accident.

Reply:
Since it was decided to form a linter report in the console, a file for generating an html report for the remark linter is not needed. I need to return setup.py to its state in the develop branch?